solutions/kubernetes/functions.sh

```diff
@@ -70,9 +72,18 @@ function passwordless_ssh() {
     local PASS=$3
     ping -c1 -W1 -q $NODE
     check_status
-    yum list pdsh -q || yum install epel-release -y
-    yum install sshpass openssh-clients pdsh -y
-    check_status "$NODE: Package installation failed"
+
+    if [[ "$ID" == "rocky" || "$ID" == "centos" ]]; then
```

Comment:
how are we getting this `$ID`?

Reply:
https://github.com/Seagate/cortx-re/blob/33a33b8e745008237cd15e308237385b041dbe6c/solutions/kubernetes/functions.sh#L21